docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,481 @@
+   "id": "8f62da67",
+   "metadata": {},
+   "source": [
+    "The expected format of these `labels` and `predictions` is modeled after what common models like mmdet and detectron2 expect. The 5 possible class labels in our dataset are car, chair, cup, person, traffic light. These classes are zero-indexed and represented with an integer 0,1,...,4. \n",
+    "\n",
+    "The 5 possible class labels in our dataset are: car, chair, cup, person, traffic light. These classes are zero-indexed and represented with an integer 0,1,...,4.\n",
+    "\n",
+    "`labels` is a list of dictionaries where `labels[i]` is a dictionary containing key `labels`, which is a list of class labels corresponding key `bboxes`, which is a numpy array of bounding boxes. Each bounding box in `labels[i]['bboxes']` is in the format is in the format ``[x1,y1,x2,y2]`` where `(x1,y1)` corresponds to the bottom-left corner of the box and `(x2,y2)` corresponds to the top-right. An example output of `labels[i]` is below:\n",
+    "\n",
+    "```ipython\n",
+    "{'bboxes': array([[ 26.5 , 215.25,  88.  , 229.75],\n",
+    "        [116.5 , 189.57, 166.5 , 215.07],\n",
+    "        [241.95, 180.42, 293.32, 225.82]], dtype=float32),\n",
+    " 'labels': array([1, 1, 1]), \n",
+    " ...\n",
+    " 'seg_map': '000000037777.png'} \n",
+    "```\n",
+    "\n",
+    "`predictions` is a list of numpy arrays where `predictions[i]` is of shape `(M,)` where `M` is the number of classes and `predictions[i][m]` is of shape `(L,5)` where `L` is the number of bounding boxes for class `k` and the five columns correspond to ``[x1,y1,x2,y2,pred_prob]`` returned by the model. Here `pred_prob` is the model confidence in the predicted label. Our example uses `k == 5` classes which is why `predictions[0].shape = (5,)`.\n",
+    "\n",
+    "An example output of `predictions[i]` is below:\n",
+    "\n",
+    "```ipython\n",
+    "array([array([], shape=(0, 5), dtype=float32),\n",
+    "       array([[2.41300339e+02, 1.77116043e+02, 2.97747314e+02, 2.28405350e+02,\n",
+    "               1.15757108e-01],\n",
+    "              [8.42839813e+01, 1.88506973e+02, 1.69366867e+02, 2.28179962e+02,\n",
+    "               1.06976844e-01]], dtype=float32)                               ,\n",
+    "       array([[2.09936218e+02, 1.22989265e+02, 2.15952652e+02, 1.33577789e+02,\n",
+    "               2.66048908e-01],\n",
+    "              [3.35775848e+02, 5.53170547e+01, 3.52000000e+02, 7.79105530e+01,\n",
+    "               6.99099600e-02]], dtype=float32)                               ,\n",
+    "       array([], shape=(0, 5), dtype=float32),\n",
+    "       array([], shape=(0, 5), dtype=float32)], dtype=object)\n",
+    "```\n",
+    "\n",
+    "Once you have pred_probs and labels in the appropriate formats, you can **find label issues with cleanlab for any object detection dataset**!\n",
+    "\n",
+    "For our specific example visualized above, the `label` and `prediction` is shown below. "
```

Comment:
Do we need to show an example of `labels` and `predictions` twice? It has been shown one in the md block above and here we are printing again.

Also typo:

```suggestion
    "For our specific example visualized above, the `labels` and `predictions` is shown below. "
```